### PR TITLE
Documented the options for changing the iDrac network setting

### PR DESCRIPTION
**Pull request by tannoa2**

### CODE REVIEW

**Comment by tannoa2** [comment]
@RackHD/veyron

File: lib/task-data/schemas/dell-racadm-control.json [hunk 1]
```diff
@@ -18,6 +18,18 @@
             "description": "Specify if reboot after racadm control tasks are done",
             "type": "boolean"
         },
+        "ip": {
```
Comment (srinia6) [inline lib/task-data/schemas/dell-racadm-control.json]: @tannoa2  You might want to reference these from properties.

Comment (srinia6) [inline lib/task-data/schemas/dell-racadm-control.json]: NVM, got it!

Comment (pengz1) [inline lib/task-data/schemas/dell-racadm-control.json]: Same question: why those schema is not added to properties? And support you should only need one.

File: lib/task-data/schemas/dell-racadm-control.json [hunk 2]
```diff
@@ -18,6 +18,18 @@
...
+            "description": "The ip address of the new iDrac  network setting",
+            "type": "string"
```
Comment (brianparry) [inline lib/task-data/schemas/dell-racadm-control.json]: @tannoa2 You should add `format ipv4`.

File: lib/task-data/schemas/dell-racadm-control.json (hunk 1 above)
Comment (pengz1) [inline lib/task-data/schemas/dell-racadm-control.json]: Schema in a file is usually defined in "definitions" and then used in "properties". If items are not specified in "properties" they won't take effect.
I saw you defined those items for IP, netmask, gateway, however I didn't see you use those definitions.

Comment (pengz1) [inline lib/task-data/schemas/dell-racadm-control.json]: You should add some unit test cases to verify it.

**Comment by anhou** [comment]
@tannoa2  is this PR still valid?

**Pull request closed without merging**